test/Microsoft.AspNetCore.SignalR.Tests/ServerFixture.cs

```diff
@@ -42,11 +43,13 @@ public void ConfigureServices(IServiceCollection services)
                 services.AddSockets();
                 services.AddSignalR();
                 services.AddEndPoint<EchoEndPoint>();
+                services.AddEndPoint<TestEndPoint>();
             }
 
             public void Configure(IApplicationBuilder app, IHostingEnvironment env)
             {
                 app.UseSockets(options => options.MapEndPoint<EchoEndPoint>("echo"));
+                app.UseSockets(options => options.MapEndPoint<TestEndPoint>("testEndPoint"));
```

Comment:
We should be consistent with naming. That means `test` for this, or change the URLs for the others.